src/olympia/addons/models.py

```diff
@@ -2170,6 +2171,11 @@ class AddonGUID(ModelBase):
     guid = models.CharField(max_length=255, null=False, db_index=True)
     addon = models.OneToOneField(
         Addon, null=False, on_delete=models.CASCADE, unique=True)
+    hashed_guid = models.CharField(max_length=100, null=True)
```

Comment:
We could maybe lower the `max_length` but 🤷 

Also, I think we have to use `null=True` because we're adding a new field to an existing table and we'll run the backfill after the creation of this new column in the DB. @diox should we file an issue to set this field to `null=False` later?

Reply:
Yes.